scenario_player/main.py

```diff
@@ -559,7 +559,7 @@ def smoketest(eth_client: EthClient):
                 config_file = create_smoketest_config_file(setup, datadir)
 
                 keystore_file = os.path.join(setup.args["keystore_path"], "keyfile")
-                password_file = setup.args["password_file"].name
+                password_file = setup.args["password_file"]
```

Comment:
Is this correct?
If it's now a `Path` then this should probably be `str(setup.args["password_file"])`, or the downstream needs to be changed to work with Path instances.

Reply:
The downstream (`get_password( )` ) uses `open(password_file, 'r' )` to read in the file, which should be fine to accept `Path` instances?

Reply:
Hm yeah since the `__fspath__` protocol was introduced it should indeed work.